basis/html/templates/chloe/compiler/compiler.factor

```diff
@@ -74,17 +79,38 @@ SYMBOL: string-context?
 
 ERROR: tag-not-allowed-here ;
 
+CONSTANT: self-closing-tags {
```

Comment:
It actually seems weird to have both words, honestly. I knew about `in?` calling `member?`, but figured that you were aiming not to imply that I was using a literal set. But I'm not clear if we even ever care.